docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -0,0 +1,255 @@
+---
+title: "What's new in accessibility in the .NET Framework"
+ms.custom: "updateeachrelease"
+ms.date: "10/13/2017"
+ms.prod: ".net-framework"
+ms.technology: 
+  - "dotnet-clr"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "what's new [.NET Framework]"
+author: "rpetrusha"
+ms.author: "ronpet"
+manager: "wpickett"
+---
+
+# What's new in accessibility in the .NET Framework
+
+The .NET Framework aims at making applications more accessibile for your users. Accessibility features allow an application to handle a range of devices and form factors seamlessly. Starting with the .NET Framework 4.7.1, the .NET Framework includes a large number of accessibility improvements that allow developers to create accessible applications. 
+
```

Comment:
Could you say something about supporting Assistive Technology apps here?  It's not so much about form factors as about providing the right experience for users of assistive technology.